packages/cli/test/helpers/buildpacks/buildpack-installations-stub.ts

```diff
@@ -1,4 +1,5 @@
-import {Scope} from 'nock'
+// eslint-disable-next-line node/no-extraneous-import
```

Comment:
Not sure about this. Ideally we use the same version of nock as fancy-test from @oclif/test. 
We could also specify the version in cli/package.json, disabling this lint rule seems like the better choice.